File: plugins/devkit/devkit-java-tests/testData/inspections/incorrectServiceRetrieving/AppLevelServiceAsProjectLevel.java
import com.intellij.openapi.components.Service;
import com.intellij.openapi.project.Project;

@Service(Service.Level.APP)
final class MyService {
  void foo(Project project) {
    MyService service1 = project.<error descr="The application-level service is retrieved as a project-level service">getService</error>(MyService.class);
    MyService service2 = project.<error descr="The application-level service is retrieved as a project-level service">getServiceIfCreated</error>(MyService.class);
  }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/retrievingLightService/AppLevelServiceAsProjectLevel.kt
import com.intellij.openapi.components.Service;
import com.intellij.openapi.project.Project;

@Service(Service.Level.APP)
class MyService {
  fun foo(project: Project) {
    project.<error descr="[NEW_INFERENCE_NO_INFORMATION_FOR_PARAMETER] Not enough information to infer type variable T">getService</error>(MyService::class.<error descr="[UNRESOLVED_REFERENCE] Unresolved reference: java">java</error>)
  }
}
